lib/compare_task.rb

```diff
@@ -94,9 +97,14 @@ def render_comparison(description1, description2, scopes, options = {})
         identical = false if description1[scope] || description2[scope]
       end
     end
-
     output = "Compared descriptions are identical.\n" + output if identical && common_scopes
 
+    if !identical_scopes.empty?
+      output += "Following #{Machinery::pluralize(
+      identical_scopes.count, "scope is", "scopes are")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Prefer single-quoted strings inside interpolations.